### PR TITLE
hotfix-search

### PR DESCRIPTION
**Pull request by nensidosari**
## Changes
- fixes https://dailydotdev.slack.com/archives/C01L0QXQJNB/p1714138510191369

### Preview domain
https://hotfix-search.preview.app.daily.dev

### CODE REVIEW

**Review by omBratteng — APPROVED** [review]

File: packages/shared/src/components/layout/MainLayoutHeader.tsx [hunk 1]
```diff
@@ -99,7 +99,7 @@ function MainLayoutHeader({
       <SearchPanel
         className={{
           container: classNames(
-            'laptop:z-auto left-0 top-0 z-header mx-auto bg-background-default py-3 tablet:left-16 laptop:left-0 laptop:top-14 laptop:bg-transparent',
+            'laptop:z-index laptop:top:14 left-0 top-0 z-header mx-auto bg-background-default py-3 tablet:left-16 laptop:left-0 laptop:bg-transparent',
```
[the remark this answers is not in the document]
Comment (omBratteng) [reply packages/shared/src/components/layout/MainLayoutHeader.tsx:102]: Don't think that `laptop:z-index` do anything 🤔

[the remark this answers is not in the document]
Comment (nensidosari) [reply packages/shared/src/components/layout/MainLayoutHeader.tsx]: doesn't do anything, removed

**Review by sshanzel — APPROVED** [review]
🚀

File: packages/shared/src/components/search/SearchContainer.tsx [hunk 2]
```diff
@@ -20,7 +20,7 @@ export function SearchContainer({
   chunk,
 }: SearchContainerProps): React.ReactElement {
   return (
-    <div className="m-auto grid w-full max-w-screen-laptopL grid-cols-1 gap-y-6 py-8 laptop:grid-cols-3">
+    <div className="m-auto grid w-full max-w-screen-laptopL grid-cols-1 gap-y-6 py-8 laptop:grid-cols-3 mt-14 laptop:mt-auto">
```
Comment (capJavert) [inline packages/shared/src/components/search/SearchContainer.tsx:23]: Check mobile because it is separate styling then desktop. 🙏

Comment (nensidosari) [reply packages/shared/src/components/search/SearchContainer.tsx:23]: yes have checked, this was done to accommodate mobile view, see images on comments above

**Pull request merged**